pulumi/src/api.ts

```diff
@@ -17,11 +17,12 @@ export interface ApiConfig {
 }
 
 export interface DeployApiArgs {
-  app: string
-  asset: string
+  appName: string
+  coinstack: string
+  assetName: string
```

Comment:
want to make the same naming/label changes for `statefulService` as well?

Reply:
sure